exoservice-javascript/bin/setup

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 
 yarn install
+webpack
```

Comment:
Please create an executable file `bin/build` that runs `webpack`. Then call `build` here. This way a developer can simply run `build` on the command line and it always does the right thing in each repo.